package.json

```diff
@@ -35,6 +37,88 @@
     "prettier": "^2.4.1",
     "pretty": "^2.0.0",
     "typescript": "^4.5.2",
-    "@types/jest": "^27.0.2"
+    "wireit": "^0.7.1"
+  },
+  "wireit": {
+    "lint": {
+      "files": ["./examples", "./scripts", "./packages/spectacle/src", "!**/node_modules", "!**/es", "!**/lib", "!**/dist"],
```

Comment:
You'll notice that some of these commands have a `files` field, and others don't. My understanding is that the `files` field is used for watching, and to generate a checksum for caching. Therefore, I added the `files` field in scripts that I thought it'd make sense to run in `--watch` mode, or ones that will run in CI that it makes sense to have proper caching on.